src/style.css

```diff
@@ -1 +1,64 @@
-/* Styles go here */
+* {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+header {
+  display: flex;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.menu {
+  display: flex;
+  list-style: none;
+}
+
+.list {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-right: 20px;
+}
+
+.link {
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  align-items: center;
+  font-weight: 500;
+  text-transform: uppercase;
+  display: flex;
+  margin-top: 23px;
+  height: 60px;
```

Comment:
use paddings instead of margin and height

Reply:
done